src/openforms/payments/contrib/ogone/tests/test_client.py

```diff
@@ -1,14 +1,62 @@
+import os
+from pathlib import Path
+
 from django.test import TestCase
 
+import requests
+
+from openforms.utils.tests.vcr import OFVCRMixin
+
 from ..client import OgoneClient
 from ..plugin import RETURN_ACTION_PARAM
 from .factories import OgoneMerchantFactory
 
 
-class OgoneClientTest(TestCase):
+class OgoneClientTest(OFVCRMixin, TestCase):
+    VCR_TEST_FILES = Path(__file__).parent / "files"
+
+    def setUp(self):
+        super().setUp()
+        self.merchant = OgoneMerchantFactory(
+            pspid=os.getenv("OGONE_PSPID", "maykinmedia"),
+            sha_in_passphrase=os.getenv("OGONE_SHA_IN", "placeholder_sha_in"),
+            sha_out_passphrase=os.getenv("OGONE_SHA_OUT", "placeholder_sha_out"),
+        )
```

Comment:
aaah that makes sense!I was wondering why we use create since this can be considered as obvious!

Reply:
Got misled again here: wanted to do a `build`, but got used to the Django model instantiation (which does the same). It would have been great if factory boy used the `build` strategy on instantiation instead of `create`